docs/ja/core-concepts/ngsiv2-vs-ngsild.md

```
@@ -5,27 +5,27 @@ outline: deep
 ---
 # NGSIv2 / NGSI-LD プロトコル分離
 
-GeonicDB は、単一の Context Broker で NGSIv2 と NGSI-LD の両方の API をサポートしています。両方の API は統一された内部ストレージ形式を共有していますが、**エンティティはプロトコルごとに分離されています** -- NGSIv2 で作成されたエンティティは NGSIv2 でのみアクセス可能で、NGSI-LD についても同様です。
+GeonicDB は単一の Context Broker 内で NGSIv2 と NGSI-LD の両方の API をサポートしています。両 API は統一された内部ストレージ形式を共有していますが、**エンティティはプロトコルごとに分離されています** -- NGSIv2 で作成されたエンティティは NGSIv2 でのみアクセス可能であり、NGSI-LD も同様です。
 
 ## 目次
 
 - [概要](#概要)
 - [統一された内部形式](#統一された内部形式)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)